single-line-diagram/single-line-diagram-cgmes/single-line-diagram-cgmes-layout/src/main/java/com/powsybl/sld/cgmes/layout/LayoutToCgmesExtensionsConverter.java

```diff
@@ -252,6 +254,20 @@ private void applyLayoutOnVscConverterStation(Node node, VoltageLevel voltageLev
         }
     }
 
+    private void applyLayoutOnLccConverterStation(Node node, VoltageLevel voltageLevel, String diagramName, OffsetPoint offsetPoint) {
+        FeederNode lccNode = (FeederNode) node;
+        LccConverterStation lccConverterStation = voltageLevel.getConnectable(lccNode.getId(), LccConverterStation.class);
+        if (lccConverterStation != null) {
+            LineDiagramData<LccConverterStation> lccDiagramData = LineDiagramData.getOrCreateDiagramData(lccConverterStation);
+            int danglingLineSeq = getMaxSeq(lccDiagramData.getPoints(diagramName)) + 1;
+            DiagramPoint vcsPoint = offsetPoint.newDiagramPoint(lccNode.getX(), lccNode.getY(), danglingLineSeq);
```

Comment:
A very minor remark: could you please rename `vcsPoint` in `lccPoint` just to be consistent with the notations?
By the way, I am under the impression that `vcs` should be `vsc` instead in `applyLayoutOnVscConverterStation` for example.